server_addon/houdini/client/ayon_houdini/api/plugin.py

```diff
@@ -14,9 +14,14 @@
     CreatedInstance,
     AYON_INSTANCE_ID,
     AVALON_INSTANCE_ID,
+    load
 )
 from ayon_core.lib import BoolDef
 from .lib import imprint, read, lsattr, add_self_publish_button
+import pyblish.api
```

Comment:
This import should be after `import hou`

Reply:
do you mean _shouldn't_ ? 
because it's already after `import hou`

Reply:
It's about PEP ordering of imports.
1. dunder imports
2. built-in imports
3. 3rd party imports
4. from the package itself
5. relative imports

Reply:
I fixed it and all other files too. Thanks.
https://github.com/ynput/ayon-core/pull/536/commits/a47c329273bff64b0761e1cb6b2aaae1ead8ad6c